sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -770,6 +976,36 @@ def _joint_states_callback(self, joint_state):
             self._joints_effort = {n: v for n, v in
                                    zip(joint_state.name, joint_state.effort)}
```

Comment:
side note -> this can be just self._joints_effort = {zip(joint_state.name, joint_state.effort)}